src/compilerWrapper.ts

```diff
@@ -586,6 +594,43 @@ export function getStructDeclaration(astRoot, dependencyAsts): Array<StructEntit
   }).flat(1);
 }
 
+
+
+/**
+ * 
+ * @param astRoot AST root node after main contract compilation
+ * @param dependencyAsts AST root node after all dependency contract compilation
+ * @returns all defined Library of the main contract and dependent contracts
```

Comment:
Library -> libraries

Use an alias, otherwise confusing.
```
type LibraryEntity = StructEntity

Array<StructEntity>
->
Array<LibraryEntity>
```